x-pack/plugins/ml/public/application/jobs/jobs_list/components/delete_job_modal/delete_job_modal.tsx

```diff
@@ -133,6 +138,17 @@ export const DeleteJobModal: FC<Props> = ({ setShowFunction, unsetShowFunction,
                       jobsCount: jobIds.length,
                     }}
                   />
+                  <EuiSpacer />
+                  <EuiSwitch
+                    label={i18n.translate(
+                      'xpack.ml.jobsList.deleteJobModal.deleteUserAnnotations',
+                      {
+                        defaultMessage: 'Also delete annotations.',
```

Comment:
Looking at the text we have in the modal when deleting DFA jobs, maybe this could just be `Delete annotations` ?